pkg/gadget-service/store/k8s-configmap-store/configmapstore.go

```diff
@@ -234,6 +234,17 @@ func (s *Store) handleErr(err error, key string) {
 func (s *Store) CreateGadgetInstance(ctx context.Context, req *api.CreateGadgetInstanceRequest) (*api.CreateGadgetInstanceResponse, error) {
 	log.Debugf("create gadget instance: %+v", req.GadgetInstance.GadgetConfig)
 
+	// Check whether the gadget instance with the same name already exists
+	instances, err := s.ListGadgetInstances(ctx, &api.ListGadgetInstancesRequest{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to list gadget instances: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("listing gadget instances: %w", err)
```

(to make the semgrep on the CI happy)

Reply:
Done!